clients/itwin/src/ImsAuthorizationClient.ts

```diff
@@ -7,13 +7,15 @@
  */
 
 import { Client } from "./Client";
+import { Config } from "@bentley/bentleyjs-core";
 
 /** @beta */
 export class ImsAuthorizationClient extends Client {
   public static readonly searchKey: string = "IMSOpenID";
 
   public constructor() {
     super();
+    this._url = Config.App.query("imjs_itwin_platform_authority");

```

Comment:
if the env var is undefined, do you need to explicitly fallback to imsoidc here? or is that done elsewhere?